tutorials/fmc/fmc.h

```diff
@@ -0,0 +1,32 @@
+#include "libs/adc/api.h"
+#include "libs/timer/api.h"
+
+adc_pin_e adc_pin = ADC4;
+
+void timer0_callback(void);
+
```

Comment:
I recommend leaving comments about what these timers are doing and how you calculated the output compare match values and what they do. For both timers.